acceptance_tests/features/validation.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const translation = require('./../../apps/complaints/translations/en/default');
+
+Feature('Error messages appear when going through the form when step invalid');
+
+Scenario('Error message appears on the /who step when Save and continue is clicked without an option selected', (I) => {
+  I.amOnPage('/');
+  I.seeInCurrentUrl('/who');
+  I.see(translation.pages['personal-contact-details'].header);
+  I.click(translation.buttons.continue);
+  I.see(translation.validation['who-radio'].required);
+});
```

Comment:
can we move this to the step? Grouping acc tests by step rather than type of test would make more sense